packages/horizon/contracts/interfaces/IBridgeEscrow.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity 0.8.26;
+
+/**
+ * @title IBridgeEscrow
+ */
+interface IBridgeEscrow {
```

Comment:
BridgeEscrow only exists on the L1 protocol, so I think we could omit it here and in GraphDirectory? Since the L2 contracts won't have an entry for it, and we won't deploy Horizon on L1.